tests/acceptance/features/apiSharingNg1/sharedByMe.feature

```diff
@@ -3916,13 +3916,86 @@ Feature: resources shared by user
         }
       }
       """
+    When user "Brian" lists the shares shared with him using the Graph API
+    Then the HTTP status code should be "200"
+    And the JSON data of the response should match
```

Comment:
its interesting you are mixing sharebyme and sharewithme tests. 
@saw-jan are you ok with this.
tests/acceptance/features/apiSharingNg1/sharedWithMe.feature this file is dedicated to share-with-me 
I vote not to mix those if we are mixing then mix others also because it will only create confusion in future

Reply:
let's move this kind of tests to own feature file (enable/disable role). this file is already huge

Reply:
Previously it was discussed to have it covered in single scenario (https://github.com/owncloud/ocis/pull/10752#discussion_r1881351989). Yeah we can have a separate feature file for these scenarios to prevent the confusion. :+1: